packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -126,18 +129,36 @@ export class MetamaskPage implements WalletPage {
     chainId: number,
     tokenSymbol: string,
     blockExplorer = '',
+    isClosePage = false,
```

Comment:
how can i use it for tests?

Reply:
Now we use that param in the test of the L2 Unsupported banner, where before each test we add the new network